db/migrate/20171101081636_update_journal_category.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateJournalCategory < ActiveRecord::Migration[5.1]
+  def change
+    Journal.where(category: 'file').each do |journal|
+      journal.update(category: 'feedback')
+    end
```

Comment:
You can also do `Journal.where(category: 'file').update_all(category: 'feedback')` to run everything in one query, but this will skip validations etc. so just FYI :)